crates/nu-command/src/commands/str_/from.rs

```diff
@@ -127,6 +127,11 @@ pub fn action(
                 }
             }
             Primitive::Decimal(dec) => format_decimal(dec.clone(), digits, group_digits),
+            Primitive::String(a_string) => a_string.to_string(),
+            Primitive::Boolean(a_bool) => a_bool.to_string(),
+            Primitive::Date(a_date) => a_date.format("%c").to_string(),
+            Primitive::FilePath(a_filepath) => a_filepath.as_path().display().to_string(),
+            Primitive::Filesize(a_filesize) => a_filesize.to_string(),
```

Comment:
Line #134 turns the size into an int and then makes a string of the int, rather than turning what we'd print as the filesize into a string.

Reply:
Updated to format filesize with units. Also renamed the friggin `b::` DebugDocBuilder to `DbgDocBldr` which is why so many files are changed now. Who's idea was to name it `b` - lolz.

With `filesize_format = "KiB"` 
```
ls | where name == 'Cargo.lock' | get size | str from
152.9 KiB
```